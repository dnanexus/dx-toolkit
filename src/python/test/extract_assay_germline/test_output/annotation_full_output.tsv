allele_id	gene_name	gene_id	feature_id	consequences	putative_impact	hgvs_c	hgvs_p
