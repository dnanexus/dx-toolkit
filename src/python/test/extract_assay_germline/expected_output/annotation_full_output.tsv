allele_id	gene_name	gene_id	feature_id	consequences	putative_impact	hgvs_c	hgvs_p
18_47408_G_A	TUBB8P12	ENSG00000173213	ENST00000308911.8	synonymous_variant	LOW	c.1317C>T	p.Ala439Ala
